src/Components/Publishing/Display/DisplayAd.tsx

```diff
@@ -26,20 +26,28 @@ export const DisplayAd: SFC<DisplayAdProps> = props => {
     ...otherProps
   } = props
 
-  if (!displayNewAds) {
+  const [width, height] = adDimension.split("x").map(a => parseInt(a))
+  const [isAdEmpty, setAdEmpty] = useState(false)
+
+  const ad = (
+    <GPT
+      adUnitPath={`/21805539690/${adUnit}`}
+      targeting={targetingData}
+      slotSize={[width, height]}
+      onSlotRenderEnded={event => {
+        setAdEmpty(event.isEmpty)
+      }}
```

Comment:
Not sure if it's actually needed but we may want to look into adding the `collapseEmptyDiv` option here 
See docs: https://github.com/nfl/react-gpt/blob/master/docs/api/ReactGPT.md#props

Reply:
I saw that option, the reason I didn't elect to go that route was that the ad's parent div has a fixed height so it will display regardless of the ad div is collapsed (or set to display:none), so we need a way to remove the parent container.